doc/source/cluster/kubernetes/examples/rayjob-batch-inference-example.md

```diff
@@ -0,0 +1,190 @@
+(rayjob-batch-inference-example)
+
+# RayJob Batch Inference Example
+
+This page demonstrates how to use the RayJob custom resource to run a batch inference job on a Ray cluster.
+
+We will use an image classification workload.  The example is based on <https://docs.ray.io/en/latest/data/examples/huggingface_vit_batch_prediction.html>. Please see that page for a full explanation of the code.
+
+## Prerequisites
+
+You must have a Kubernetes cluster running and `kubectl` configured to use it, and GPUs available.  We provide a brief tutorial for setting up the necessary GPUs on Google Kubernetes Engine (GKE), but you can use any Kubernetes cluster with GPUs.
+
+## Deploy KubeRay
+
+Make sure your KubeRay operator version is at least v0.6.0.
+
+The latest released KubeRay version is recommended.
+
+For installation instructions, please follow [the documentation](../deploy/installation.md).
+
+## Step 0: Create a Kubernetes cluster on GKE (Optional)
```

Comment:
Can we reuse `user-guides/gcp-gke-gpu-cluster.md`?

Reply:
Ideally yes, let me unify these docs in a followup PR.  It will take some time to go through and test it end to end again, since we need to make sure the machine type and resource requests don't result in CUDA out of memory errors.